src/log.ts

```diff
@@ -60,6 +60,7 @@ export interface LogOptions {
   removeCircular?: boolean;
   maxEntrySize?: number; // see: https://cloud.google.com/logging/quotas
   jsonFieldsToTruncate?: string[];
+  defaultWriteDeleteCallback?: ApiResponseCallback;
```

Comment:
Is this global callback only for WriteDelete? I thought you want to have a global callback for all methods in Log class in manual layer.

Reply:
It is for `write()` and `delete()` APIs which are the only one's used to update logs. Just wanted to distinguish it to make sure we do not confuse developers

Reply:
a small correction:`delete()` does not update logs, it deletes all log entries for a provided log name as a result of the delete log operation while `write()` creates one or more log entries.

i am not sure about the "global" scope. the scope is bound by the `Log` instance, isn't it?

Reply:
> 

Based on user's feedback in a bug, the suggestion was: "I assume a lot of users will want to set up some sort of global error catching for this particular error."
Do you have specific proposal in mind how to fix this?